js/scripts.js

```diff
@@ -14,7 +14,35 @@ $(document).ready(function(){
 
     ui : {
 
+      lazyLoadFooter : function(target,offset){
+
+        var $target = $(target),
+            targetLoc = $target.offset().top,
+            footerLazyLoadTriggered = false;
+
+            if(!footerLazyLoadTriggered){
+
+              $(window).scroll(function(e){
+
+                var scrollPos = $(this).scrollTop(),
```

Comment:
Should cache `$(window)` as `$window` and use it here as well instead of `$(this)` (since there is only one window object).